README.md

```diff
@@ -60,6 +60,17 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 - **How to sign up**: [Join #Orbtoberfest](https://circleci-public.github.io/hacktoberfest/#/) and submit your PRs to this [Google Form](https://docs.google.com/forms/d/1-MbzLy0m1eRLjeye9wu76fYkzcX5A3PvgKd09EhVObY/viewform).
 - **Notes**: View full details above. Make pull requests to any open-source "orbs" for CircleCI, or create your own. When complete submit PRs to the Google form to claim swag.
 
+### G
+
+### **Gatsby**
+
+- **Swag**: T-shirt, Stickers, Socks, Hats, Pants
+- **Requirements**:
+  - 1 merged pull requests: Level 1 swag
+  - 5 merged pull requests: Level 2 swag
+- **How to sign up**: Just submit your PRs to [Gatsby] (https://github.com/gatsbyjs/store.gatsbyjs.org) and log into their [store](https://store.gatsbyjs.org/) with your GitHub account.
```

Comment:
Be sure to not have a space between the `[brackets](parenthesizes)`, or else the link is not formatted correctly when Jekyll builds the page.

```suggestion
- **How to sign up**: Just submit your PRs to [Gatsby](https://github.com/gatsbyjs/store.gatsbyjs.org) and log into their [store](https://store.gatsbyjs.org/) with your GitHub account.
```

Please check your contribution as this appears in multiple places. :+1: